constraint/pkg/client/drivers/k8scel/driver.go

```diff
@@ -84,7 +84,7 @@ func (d *Driver) AddTemplate(_ context.Context, ct *templates.ConstraintTemplate
 		return err
 	}
 	vapVars = append(vapVars, vapVarsSuffix...)
-	filterCompiler, err := cel.NewCompositedCompiler(environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion()))
+	filterCompiler, err := cel.NewCompositedCompiler(environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion(), false))
```

Comment:
In k8s code I see places that defaults it to true?

		// strictCost is always true to enforce cost limits.
		celEnvironmentSet: environment.MustBaseEnvSet(environment.DefaultCompatibilityVersion(), true),

Reply:
`false` here means that we are not enforcing `strictCost` calculation for CEL, which was the default behavior before this change was introduced.

Reply:
We should adopt whatever K8s is doing, since this is the release that will make our engine beta... we can always loosen the cost calculations, tightening them may break backwards compatibility.

Reply:
updated to follow K8s.